extensions/amp-story-360/0.1/amp-story-360.js

```diff
@@ -444,6 +451,28 @@ export class AmpStory360 extends AMP.BaseElement {
     });
   }
 
+  /**
+   * Ensures user is facing a specified point of interest.
+   * @private
+   */
+  maybeSetGyroscopeDefaultHeading_() {
+    if (this.isOnActivePage_ && this.gyroscopeControls_ && this.isReady_) {
+      const defaultHeading = parseFloat(
+        this.element.getAttribute('heading-end') ||
+          this.element.getAttribute('heading-start')
+      );
+      if (defaultHeading) {
+        const offsetHeading =
+          90 + parseFloat(defaultHeading) + this.orientationAlpha_;
+        this.renderer_.setImageOrientation(
+          offsetHeading,
```

Comment:
I know I told you to do this, but now I realize we should theoretically offset camera orientation rather than image orientation, because the user's heading value is in the camera referential, not the image one. That said, it would make almost no visible difference unless the panorama was captured with a very tilted base. It's an edge case, but if/when that happens the correction might be completely wrong. Not sure if it's worth the trouble though, because it's probably much less easy to implement.

Reply:
Good call. 
I implemented this to offset the camera orientation. Please let me know what you think.

Reply:
LGTM, thanks!